lib/premailer/premailer.rb

```diff
@@ -157,6 +157,7 @@ module Warnings
   # @option options [Array(String)] :css Manually specify CSS stylesheets.
   # @option options [Boolean] :css_to_attributes Copy related CSS attributes into HTML attributes (e.g. background-color to bgcolor)
   # @option options [String] :css_string Pass CSS as a string
+  # @option options [Boolean] :rgb_to_hex Convert RBG to Hex colors, default false
```

Comment:
I think we should name it `rgb_to_hex_attributes` because it only happens for colors that get copied into attribute fields (bgcolor, etc.), not other CSS.

This whole issue is probably a bugfix so default `true` would also seem sensible. Eventually it might not need to be controlled with the option at all but it's good to have it anyway just in case.